trackman/cli.py

```diff
@@ -4,7 +4,7 @@
 from . import app, db_utils, lib, pubsub, tasks
 
 
-@app.cli.command()
+@app.cli.command('init_embedded_db')
```

Comment:
I'm not sure this is necessary. It should use the function name by default, no?

Reply:
It didn't on my machine for some reason.